### PR TITLE
🚮 Remove notion of a lite viewer

### PR DESCRIPTION
**Pull request by alanorozco**
We would previously allow viewers to opt-in to a "lite" document by setting the query parameter `amp_lite`.

In practice, this signal was used sparingly—only to disable video autoplay.

Since viewers almost never ask for a lite document, we remove this notion.

### CODE REVIEW

File: src/service/video-manager-impl.js [hunk 1]
```diff
@@ -459,10 +459,7 @@ class VideoEntry {
 
     // eslint-disable-next-line jsdoc/require-returns
     /** @private @const {function(): !Promise<boolean>} */
-    this.supportsAutoplay_ = () => {
-      const {win} = this.ampdoc_;
-      return VideoUtils.isAutoplaySupported(win, getMode(win).lite);
-    };
+    this.isAutoplaySupported_ = () => VideoUtils.isAutoplaySupported(this.win);
```
Comment (samouri) [inline src/service/video-manager-impl.js:462]: nit: is this a useful variable? the alternative being to call `import {isAutoplaySupported} from 'src/utils/video'`;

**Review by samouri — APPROVED** [review]
Fantastic

File: src/service/video-manager-impl.js (hunk 1 above)
Comment (alanorozco) [reply src/service/video-manager-impl.js:462]: `VideoUtils` is actually a class containing static methods. It was added before our test builds could stub module exports. I'll follow up with a PR that simplifies the autoplay support module.

**Review by kristoferbaxter — APPROVED** [review]

**Review by jridgewell — APPROVED** [review]
Woohoo!

**Pull request merged**